tests/Checker/AJAX_Runner_Tests.php

```diff
@@ -10,6 +10,14 @@
 
 class AJAX_Runner_Tests extends WP_UnitTestCase {
 
+	public function tear_down() {
+		// Force reset the database prefix after runner prepare method called.
+		global $wpdb;
+		$wpdb->set_prefix( 'wp_' );
```

Comment:
Can we use the `$table_prefix` global or (even better) access the original prefix in a `set_up()` method? We shouldn't assume that the regular prefix is `wp_`.

Reply:
@felixarntz I opted for the `$table_prefix` and updated both instances.